final/index.html

```diff
@@ -0,0 +1,251 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="container">
+        <header class="header">
+            <div class="header-left">
+                <h1 class="header-title">Product name</h1>
+                <ul class="header-list">
+                    <li class="header-item">Put on this page information about your product</li>
+                    <li class="header-item">A detailed description of your product</li>
+                    <li class="header-item">Tell us about the advantages and merits</li>
+                    <li class="header-item">Associate the page with the payment system</li>
+                </ul>
+            </div>
+            <div class="header-img">
+                <img src="#" alt="">
```

Comment:
можно уже вырезать изображения и делать заглушки)

Reply:
Убрал див, дал класс картинке, не знаю сам зачем див сделал)
```suggestion
                <img class="header-img" src="#" alt="">
```